Brokkr/src/main/cpp/subsystems/Claw.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "subsystems/Claw.h"
+#include <frc/smartdashboard/SmartDashboard.h>
+
+Claw::Claw(WPI_TalonFX& mClawMotor, WPI_TalonFX& mIntakeMotor)
+: ClawMotor(mClawMotor)
+, IntakeMotor(mIntakeMotor)
+{
+    mClawMotor.SetInverted(false);
+    mClawMotor.Config_kP(0, 0, 0);
+    mClawMotor.Config_kI(0, 0, 0);
+    mClawMotor.Config_kD(0, 0, 0);
+    mClawMotor.Config_kF(0, 0, 0);
+}
+
+void Claw::Periodic() 
+{
+    frc::SmartDashboard::PutBoolean("Is Claw Open", GetClawState());
+}
+
+bool Claw::GetClawState()
```

Comment:
Let's use an enumeration for `GetClawState`.

`bool` is better than a string, but an enum is a bit more expressive.

Reply:
Rather than `GetClawState`, perhaps this is `IsClawOpen`?